kanopi/kanopi-pack
# April 24 Maintenance Updates

## Description
> As a developer, I need to install security updates on a regular basis

## Acceptance Criteria
* Existing Development and Production modes work
* Watch mode successfully loads, like Production mode

## Steps to Validate
1. Use the feature branch to update your projects dependencies, add it `npm i https://github.com/kanopi/kanopi-pack.git#feature/tw29466196/april-24-maintenance`
3. Run the Development command: `npm run development`, verify it builds the bundle and continues to listen without crashing.
4. Load a page on the site using the assets 
5. Change an asset file, CSS, JS, etc and verify the command line shows the bundle rebuild (it will take about the same time production takes to build)
6. Refresh the page and verify the changes show
7. Run `fin production` and load the site in Production mode. Verify site works as expected.
